README.adoc

```diff
@@ -43,118 +43,27 @@ make all
 make _site
 ----
 
-== Structure (and how it works)
+== Structure
 
 === General
 
-There are two types of input entries to this site:
+Run the following command to update the list of CC documents in the site:
 
-. CSD document entries
+```sh
```

Comment:
Please use `[source,xxx]\n----` instead of the Markdown style of three backticks.